README.md

```diff
@@ -95,36 +95,29 @@ _docs_ subdirectory. Version-specific documentation is also at
 
 ### Verifying binaries
 
-Download directories contain a `SHASUMS256.txt` file with SHA checksums for the
-files.
+Download directories contain a `SHASUMS256.txt.asc` file with SHA checksums for the
+files and the releaser PGP signature.
 
-To download `SHASUMS256.txt` using `curl`:
+You can get a trusted keyring from nodejs/release-keys, e.g. using `curl`:
 
 ```bash
-curl -O https://nodejs.org/dist/vx.y.z/SHASUMS256.txt
+curl -fsLo "/path/to/nodejs-keyring.kbx" "https://github.com/nodejs/release-keys/raw/HEAD/gpg/pubring.kbx"
 ```
 
-To check that downloaded files match the checksum, use `sha256sum`:
+Alternatively, you can import the releaser keys in your default keyring, see
+[Release keys](#release-keys) for commands to how to do that.
 
-```bash
-sha256sum -c SHASUMS256.txt --ignore-missing
-```
-
-For Current and LTS, the GPG detached signature of `SHASUMS256.txt` is in
-`SHASUMS256.txt.sig`. You can use it with `gpg` to verify the integrity of
-`SHASUMS256.txt`. You will first need to import
-[the GPG keys of individuals authorized to create releases](#release-keys).
-
-See [Release keys](#release-keys) for commands to import active release keys.
-
-Next, download the `SHASUMS256.txt.sig` for the release:
+Then, you can verify the files you've downloaded locally
+(if you're using your default keyring, do not pass `--keyring`):
 
 ```bash
-curl -O https://nodejs.org/dist/vx.y.z/SHASUMS256.txt.sig
-```
+set -e
+set -o pipefail
 
-Then use `gpg --verify SHASUMS256.txt.sig SHASUMS256.txt` to verify
-the file's signature.
+curl -fs "https://nodejs.org/dist/${VERSION}/SHASUMS256.txt.asc" \
+| gpgv --keyring="/path/to/nodejs-keyring.kbx" --output - \
+| shasum --check --ignore-missing
```

Comment:
For the above instructions to remove `--keyring` when using the default keyring, the instructions show an error message 

> Can't check signature: No public key

and the return code is still `0` (success) so I wonder if this is suitable for unattended use, where the error might not be noticed if there is no error return code.

Reply:
```suggestion
curl -fsO "https://nodejs.org/dist/${VERSION}/SHASUMS256.txt.asc"
gpgv --keyring="/path/to/nodejs-keyring.kbx" --output SHASUMS256.txt < SHASUMS256.txt.asc
shasum --check SHASUMS256.txt --ignore-missing
```

Reply:
Yes, you are right. I didn't use `set -o pipefail` and I only pasted the curl line into my interactive bash session. Sorry for my mistake!